packages/mobile/src/app/selectors.ts

```diff
@@ -19,3 +23,13 @@ export const getLastTimeBackgrounded = (state: RootState) => {
 export const sessionIdSelector = (state: RootState) => {
   return state.app.sessionId
 }
+
+export const verificationPossibleSelector = (state: RootState) => {
```

Comment:
Should this take into account the $ required for verification itself? The ODIS min balance just dropped to 0.01 so if a user has 3 cents this will still return true though verification isn't actually possible.